scripts/run_mypy_checks.py

```diff
@@ -0,0 +1,793 @@
+    'scripts/release_scripts/cut_release_or_hotfix_branch_test.py',
+    'scripts/release_scripts/repo_specific_changes_fetcher.py',
+    'scripts/release_scripts/repo_specific_changes_fetcher_test.py',
+    'scripts/release_scripts/update_changelog_and_credits.py',
+    'scripts/release_scripts/update_changelog_and_credits_test.py',
+    'scripts/release_scripts/update_configs.py',
+    'scripts/release_scripts/update_configs_test.py',
+    'scripts/run_backend_tests.py',
+    'scripts/run_custom_eslint_tests.py',
+    'scripts/run_e2e_tests.py',
+    'scripts/run_e2e_tests_test.py',
+    'scripts/run_frontend_tests.py',
+    'scripts/run_lighthouse_tests.py',
+    'scripts/run_mypy_checks.py',
+    'scripts/run_mypy_checks_test.py',
+    'scripts/run_portserver.py',
+    'scripts/run_presubmit_checks.py',
+    'scripts/run_tests.py',
+    'scripts/script_import_test.py',
+    'scripts/scripts_test_utils.py',
+    'scripts/scripts_test_utils_test.py',
+    'scripts/servers.py',
+    'scripts/servers_test.py',
+    'scripts/setup.py',
+    'scripts/setup_gae.py',
+    'scripts/setup_gae_test.py',
+    'scripts/setup_test.py',
+    'scripts/start.py',
+    'scripts/third_party_size_check.py',
+    'scripts/typescript_checks.py',
+    'scripts/typescript_checks_test.py',
+]
+
+
+CONFIG_FILE_PATH = os.path.join('.', 'mypy.ini')
+# TODO(#13113): Change mypy command to mypy path after Python3 migration.
+MYPY_CMD = 'mypy'
+MYPY_REQUIREMENTS_PATH = os.path.join('.', 'mypy_requirements.txt')
+PYTHON3_CMD = 'python3'
+
+
+_PARSER = argparse.ArgumentParser(
+    description='Type checking script for Oppia codebase.'
+)
+
+_PARSER.add_argument(
+    '--files',
+    help='Files to type-check',
+    action='append',
+    nargs='+'
+)
+
+
+def get_mypy_cmd(files):
+    """Return the appropriate command to be run."""
+    if files:
+        cmd = [MYPY_CMD, '--config-file', CONFIG_FILE_PATH] + files[0]
+    else:
+        excluded_files_regex = (
+            '|'.join(NOT_FULLY_COVERED_FILES + EXCLUDED_DIRECTORIES))
+        cmd = [
+            MYPY_CMD, '--exclude', excluded_files_regex,
+            '--config-file', CONFIG_FILE_PATH, '.'
+        ]
+    return cmd
+
+
+def install_mypy_prerequisites():
+    """Install mypy and type stubs from mypy_requirements.txt."""
+    cmd = [PYTHON3_CMD, '-m', 'pip', 'install', '-r', MYPY_REQUIREMENTS_PATH]
+    process = subprocess.call(
+        cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
+    return process
+
+
+def main(args=None):
+    """Runs the MyPy type checks."""
+    unused_parsed_args = _PARSER.parse_args(args=args)
+
+    python_utils.PRINT('Installing Mypy and stubs for third party libraries.')
+    return_code = install_mypy_prerequisites()
+    if return_code != 0:
+        python_utils.PRINT(
+            'Cannot install Mypy and stubs for third party libraries.')
+        sys.exit(1)
+    else:
+        python_utils.PRINT(
+            'Installed Mypy and stubs for third party libraries.')
```

Comment:
```suggestion
        python_utils.PRINT('Mypy and stubs for third party libraries installed successfully.')
```

(Optional^) [I see this pattern in other places]

Reply:
The point is that I have taken a look at this. This is the thing we do normally so is it actually needed to change it?